### ISSUE TITLE
Inference in directory mode doesn't show polygons (mmdetection models)

### ISSUE THREAD

**Issue by 0ssamaak0** [0ssamaak0/labelmm#11]
Error message
```
From the working one: mmdetection/configs/solo/solo_r50_fpn_1x_coco.py
load checkpoint from local path: mmdetection/checkpoints/solo_r50_fpn_1x_coco_20210821_035055-2290a6b8.pth
Decoding .../data/imgs\camels.jpg
Time taken to annoatate img on SOLO: 613 ms
Decoding .../data/imgs\img1.jpg
Time taken to annoatate img on SOLO: 615 ms
'dict' object has no attribute 'other_data'
Decoding .../data/imgs\img1_corrupted.jpg
Time taken to annoatate img on SOLO: 328 ms
Decoding .../data/imgs\img2.jpg
Time taken to annoatate img on SOLO: 804 ms
'dict' object has no attribute 'other_data'
Decoding .../data/imgs\img3.jpg
Time taken to annoatate img on SOLO: 526 ms
'dict' object has no attribute 'other_data'
Decoding .../data/imgs\test1_Moment1.jpg
Time taken to annoatate img on SOLO: 1543 ms
'dict' object has no attribute 'other_data'
Decoding .../data/imgs\test1_Moment2.jpg
Time taken to annoatate img on SOLO: 1250 ms
'dict' object has no attribute 'other_data'
Decoding .../data/imgs\test2_Moment1.jpg
Time taken to annoatate img on SOLO: 1140 ms
'dict' object has no attribute 'other_data'
Decoding .../data/imgs\test2_Moment2.jpg
Time taken to annoatate img on SOLO: 826 ms
'dict' object has no attribute 'other_data'
```

**Comment by Shehab37** [comment]
solved in commit d4cc4f2b9ff259394c0d5b36269ac27d8f56e1ea

**Comment by Shehab37** [comment]
thanks for the notice
